mm2src/adex_cli/src/activation_scheme_db/activation_scheme_impl.rs

```diff
@@ -0,0 +1,75 @@
+use log::{debug, error, warn};
+use serde_json::{json, Value as Json};
+use std::collections::HashMap;
+
+use super::init_activation_scheme::get_activation_scheme_path;
+use crate::helpers::read_json_file;
+
+pub struct ActivationScheme {
+    scheme: HashMap<String, Json>,
+}
+
+impl ActivationScheme {
+    fn new() -> Self {
+        Self {
+            scheme: HashMap::<String, Json>::new(),
+        }
+    }
+
+    pub fn get_activation_method(&self, coin: &str) -> Option<Json> {
+        let Some(Json::Object(object)) = self.scheme.get(coin) else { return None };
+        let mut copy = json!({});
+        for (k, v) in object.iter() {
+            // WORKAROUND: serde_json::Value does not support removing key
+            if *k == "userpass" {
+                continue;
+            }
+            copy[k] = v.clone();
```

Comment:
It does support https://github.com/KomodoPlatform/atomicDEX-API/blob/371595d6c0d322e677669544aa37ec6304140fe8/mm2src/mm2_main/tests/mm2_tests/tendermint_tests.rs#L324-L330

Reply:
Shame on me!!! Thank you, Onur!
done